ng-dev/pr/config/index.ts

```diff
@@ -58,6 +58,11 @@ export interface PullRequestConfig {
    */
   validators?: PullRequestValidationConfig;
 
+  /**
+   * Optional map of paths that have extra labels to apply when PRs are created.
+   */
+  managedLabelByPath?: {[path: string]: string[]};
```

Comment:
I think the key and value should be flipped here so that we can more efficiently find matches?

I think we should have
```
{label: string]: string[]}
```

Its much more common that we would like to apply the same label to multiple different matching paths, than wanting multiple labels on a specific path.